src/resources/Validator.js

```diff
@@ -709,6 +711,31 @@ class Validator {
     return schema;
   }
 
+  applyAdvancedConditions(component, row, data) {
+    if (!component.advancedConditions || !Array.isArray(component.advancedConditions)) {
+      return;
+    }
+
+    component.advancedConditions.forEach(condition => {
+      const result = FormioUtils.checkTrigger(component, condition.trigger, row, data);
+
+      if (result) {
+        condition.actions.forEach(action => {
+          let newValue;
+          switch (action.type) {
+            case 'property':
+              FormioUtils.setActionProperty(component, action, row, data, component, result);
+              break;
+            case 'value':
+              newValue = (new Function('row', 'data', 'component', 'result', action.value))(row, data, component, result);
```

Comment:
We may want to move this to a sandbox to isolate it properly on the server.

Reply:
Yes I agree.